### PR TITLE
NCD - Next Steps - Activities

### PR DESCRIPTION
**Pull request by anvmn**
#423

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-09-11 13-30

**Comment by anvmn** [comment]
@AronNovak 

Please review. Note that this PR defines Next steps activities, and there are several @todos that are logic related, and not implmented here. They're taken care of at #459.

File: client/src/assets/scss/_new.scss [hunk 1]
```diff
@@ -3994,6 +3998,42 @@ div.page-activity.ncd {
...
+
+      .instructions {
+        margin-bottom: 0;
+      }
+    }
+
+    .form.referral {
+
+      .separator:last-of-type {
+        display: none;
```
Comment (AronNovak) [inline client/src/assets/scss/_new.scss:4033]: Isn't it something that we could solve in Elm more or less with the same easiness?

Comment (anvmn) [reply client/src/assets/scss/_new.scss:4033]: @AronNovak 

Well, what happens is that we have several blocks, each ending with separator.
We don't know which ones will apear.
So, we could modify code to insert these separators between the blocks, but it's much harder than this CSS.

**Review by AronNovak — APPROVED** [review]

**Pull request merged**